src/marqo/tensor_search/add_docs_utils.py

```diff
@@ -0,0 +1,138 @@
+"""Functions used to download and preprocess audio and video files"""
+
+from contextlib import contextmanager
+import torch
+import math
+
+# for multimodal processing
+import tempfile
+import os
+import subprocess
+from urllib.parse import urlparse
+from typing import List, Dict
+import time
+import ffmpeg
+
+from torchvision.transforms import Compose
+from marqo.core.models.marqo_index import *
+from marqo.s2_inference.s2_inference import  Modality
+
+
+class StreamingMediaProcessor:
```

Comment:
Why is this module not named after the class? As noted before utility modules are almost always a bad idea

Reply:
Thanks, I made utility modules because I did not recall that we were phasing them out. I'll rename this and try to get rid of multimodal_utils